pkgs/applications/networking/cluster/k3s/docs/ONBOARDING_MAINTAINER.md

```diff
@@ -26,7 +26,7 @@ If you cause a regression (we've all been there), you are responsible for fixing
 
 To merge code, you need to be a committer, or use the merge-bot, but currently the merge-bot only works for packages located at `pkgs/by-name/`, which means, K3s still need to be migrated there before you can use merge-bot for merging. As a non-committer, once you have approved a PR you need to forward the request to a committer. For deciding which committer, give preference initially to K3s committers, but any committer can commit. A committer usually has a green approval in PRs.
 
-K3s's committers currently are: superherointj, marcusramberg, Mic92.
+K3s's committers currently are: marcusramberg, Mic92.
```

Comment:
@marcusramberg You are still interested in maintaining k3s? I am still considering myself more of a part-time user.

Reply:
I'm kinda in the same boat, I just run a single node cluster on my desktop at the moment, accepted to join this to spread the load, so I'm not super eager to be the main driver ;-) 

Really sorry to see you go, @superherointj - thanks for all the contributions.